doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
 event:
 
     somePromise.then(function(res) {
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+'unhandledRejection':
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
+    }
+
+    var resource = new SomeResource();
+    // no .catch or .then on resource.loaded for at least a turn
+
+To deal with cases like this, which are likely false positives, you can either
```

Comment:
In the example, the loaded promise is initially in the errored state, since no resource has been loaded. This isn't a developer error; it's just part of the system. If the developer *never* loaded anything into the resource, that would be an error. But the transient errored state is not something that should be logged.